### DISCUSSION TITLE
duckdb CLI in Powershell - multi-line SQL statements

### DISCUSSION THREAD

**Discussion by codeulike** [duckdb/duckdb#12830] [Q&A]
Hi, when I use duckdb cli from Powershell, and enter a multi-line SQL statement, it works OK. However the prompt for the second and subsequent lines of the multi-line statement is a mangled character like "┬À". See screenshot:

![image](https://github.com/duckdb/duckdb/assets/999083/37beeb9e-3edc-436b-9de7-a69d2f325b11)

Presumably this is something to do with UTF8 vs ISO-8859-1. I guess I need to tell my Powershell terminal to run in UTF-8, or tell duckdb to run in ISO-8859-1. Which, and how should I do that?

NB: I have this issue both in Windows Powershell 5.1 and also Powershell 7.2.2 running in VSCode

Many thanks

**Comment by codeulike** [comment]
Solved this, by default Powershell will use the windows system locale for character encoding - its usually ansi
To set the powershell console to use UTF-8, set the following three settings:

`$OutputEncoding = New-Object System.Text.UTF8Encoding`

`[Console]::InputEncoding = New-Object System.Text.UTF8Encoding`

`[Console]::OutputEncoding = New-Object System.Text.UTF8Encoding`

(explained in detail here https://stackoverflow.com/a/49481797/22194 )

**Reply by robertdlangford** [reply]
Solves the `┬À` for me, but makes my tables look less nice 😢 

Before:
<img width="272" height="173" alt="image" src="https://github.com/user-attachments/assets/cf0ff634-8cae-4898-bfab-82c99e80f1e0" />

After:
<img width="283" height="170" alt="image" src="https://github.com/user-attachments/assets/e709d77c-49a9-4688-adb0-11965bdedcf8" />

**Reply by codeulike** [reply]
hmm, I spoke too soon - thats even worse. Is UTF-8 even what I should be aiming for here?
